addons/website_sale/models/account_move.py

```diff
@@ -12,6 +13,14 @@ class AccountMove(models.Model):
         help='Website through which this invoice was created for eCommerce orders.',
         store=True, readonly=True, tracking=True)
 
+    def _auto_init(self):
+        if not column_exists(self.env.cr, "account_move", "website_id"):
+            # Creating the column via _auto_init prevents a MemoryError in large databases.
+            # Because sale_order.website_id is a new field in this module,
+            # we can assume its value to be empty.
```

Comment:
Hello, thanks for the fix :)

Maybe explain why `create_column` actually does the trick here; I don't think the current comment would be clear enough.
```suggestion
            # Creating the column via `_auto_init` prevents a MemoryError in databases where many
            # invoices exist when `website_sale` is installed, as it skips the computation of the
            # `website_id` field.
```